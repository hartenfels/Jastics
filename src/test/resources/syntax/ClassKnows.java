public class Sample knows "music.rdf" {
  public class ObjectInner knows "abc.rdf" {}
  static class StaticInner knows "xyz.rdf" {}
}
