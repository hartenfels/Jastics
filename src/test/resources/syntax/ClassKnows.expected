public class Sample knows "share/music.rdf" {
  public class ObjectInner knows "share/abc.rdf" {
  }

  static class StaticInner knows "share/xyz.rdf" {
  }
}
